Dockerfile

```diff
@@ -9,7 +9,7 @@ ADD demo/inventory /runner/inventory
 # Install Ansible and Runner
 ADD https://releases.ansible.com/ansible-runner/ansible-runner.el8.repo /etc/yum.repos.d/ansible-runner.repo
 RUN dnf install -y epel-release && \
-    dnf install -y ansible-runner python3-pip sudo rsync openssh-clients sshpass glibc-langpack-en && \
+    dnf install -y ansible-runner python3-pip sudo rsync openssh-clients sshpass glibc-langpack-en git && \
     alternatives --set python /usr/bin/python3 && \
     pip3 install ansible && \
```

Comment:
@jladdjr I have https://github.com/ansible/ansible-runner/pull/473 open

I'm only committed to working in conjunction with your branch, not with devel, and I don't urgently need that change.

So I can close that if you get the change in here.